docs/release-notes/release-notes-0.18.0.md

```diff
@@ -421,6 +413,18 @@ bitcoin peers' feefilter values into account](https://github.com/lightningnetwor
   manage the lifecycle of the inputs.
 
 ## Breaking Changes
+
+* Previously when callng `SendCoins`, `SendMany`, `OpenChannel` and
+  `CloseChannel` for coop close, it is allowed to specify both an empty
+  `SatPerVbyte` and `TargetConf`, and a default conf target of 6 will be used.
+  This will [no longer be
+  allowed](https://github.com/lightningnetwork/lnd/pull/8422) in the next
+  release (v0.19.0) and the caller must specify either `SatPerVbyte` or
```

Comment:
cc @saubyk do you think it's feasible to enforce this in the next release, or should we wait even longer, say 0.20.0?

Reply:
Given that we realized the impact at a later stage in the game, here's my recommendation:
- Deprecate the rpcs in 0.19 and publish a new set of apis (v2/v3) which would require user to specify the fee rate
- remove the deprecated rpcs in 0.20 which would force the users to move to the new rpcs

Reply:
I think adding a new set of RPC endpoints because of one field is an overkill. Normally we'd add a deprecated flag to the existing field.